util/src/entities/MemberVerificationForm.ts

```diff
@@ -0,0 +1,33 @@
+import { Column, Entity, Index, JoinColumn, ManyToOne, PrimaryColumn, RelationId } from "typeorm";
+import { Guild } from "./Guild";
+import { BaseClassWithoutId } from "./BaseClass";
+
+export interface MemberVerificationFormField {
+	field_type: string;
+	label: string;
+	required: boolean;
+	values: string[];
+}
+
+@Entity("member_verification_forms")
+@Index(["id"], { unique: true })
+export class MemberVerificationForm extends BaseClassWithoutId {
+	@PrimaryColumn()
+	@RelationId((form: MemberVerificationForm) => form.guild)
+	id: string;
+
+	@JoinColumn({ name: "id" })
+	@ManyToOne(() => Guild, {
+		onDelete: "CASCADE",
+	})
+	guild: Guild;
+
+	@Column()
+	description: string;
+
+	@Column({ type: "simple-json" })
+	form_fields: MemberVerificationFormField[];
```

Comment:
separate this into its own table so that the questions can be shared by multiple guilds/user groups

Reply:
How should I go about doing that? 
One of the problems I've seen so far is that a few of the field types are an array of strings which can contain commas, so storing those in a simple-array would break the array as typeorm uses commas as the deliminator.
Another thing is that when adding/removing form fields, the client will send all of the fields instead of just whats been changed, but there's no way to distinguish existing ones that I can think of since the form fields don't have an id in the request.

Reply:
also that the relationship between questionnaires and guilds/roles/groups should be structured as one questionnaire to zero or more guild/role/group.